packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -1222,42 +1222,3 @@ function writeWranglerConfig() {
     "utf-8"
   );
 }
-
-export function mockKeyListRequest(
```

Comment:
we shouldn't have shared functions inside test files, because any other test file that imports the function will end up redefining all of these tests inside itself too (side effects are just groovy lol). So I moved this to its own module.

Reply:
Yes! I was thinking we should actually go further and have all these helpers in a shared place so that they can be used in other tests generally.

Reply:
Actually, yes, that makes more sense, we can consolidate the suite of helpers we seem to have built so far